app/Services/BaseService.php

```diff
@@ -92,6 +93,11 @@ public function imageName(): string
         return $this->imageName;
     }
 
+    public static function category(): string
+    {
+        return Str::title(static::$category) ?? 'Other';
```

Comment:
Could we actually just return the lowercase name here, and then run `Str::title` on it in the consuming code?